--- repositories/stable-diffusion-stability-ai/ldm/models/diffusion/ddpm.py	2022-12-29 01:21:59.977943990 -0500
+++ repositories/stable-diffusion-stability-ai/ldm/models/diffusion/ddpm.py 2023-01-05 12:58:38.284139191 -0500
@@ -900,6 +900,7 @@
         loss_simple = self.get_loss(model_output, target, mean=False).mean([1, 2, 3])
         loss_dict.update({f'{prefix}/loss_simple': loss_simple.mean()})

+        self.logvar = self.logvar.to(self.device)
         logvar_t = self.logvar[t].to(self.device)
         loss = loss_simple / torch.exp(logvar_t) + logvar_t
         # loss = loss_simple / torch.exp(self.logvar) + self.logvar
